examples/ioredis/tracer.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const opentelemetry = require('@opentelemetry/core');
+const { NodeTracerRegistry } = require('@opentelemetry/node');
+const { SimpleSpanProcessor } = require('@opentelemetry/tracing');
+const { JaegerExporter } = require('@opentelemetry/exporter-jaeger');
+
+const tracerRegistry = new NodeTracerRegistry();
+
+const exporter = new JaegerExporter({ serviceName: 'ioredis-example' });
+
+tracerRegistry.addSpanProcessor(new SimpleSpanProcessor(exporter));
+
+// Initialize the OpenTelemetry APIs to use the BasicTracer bindings
+opentelemetry.initGlobalTracerRegistry(tracerRegistry);
+
+module.exports = opentelemetry.getTracer();
```

Comment:
@dyladan should this now be `module.exports = opentelemetry.getTracerRegistry();` ?
Should the file be renamed to `tracer-registry.js` in this case?

As a side note I find this whole tracer registry/factory thing to be confusing for the end user. I wish the focus was that libs just not have broken implementations of tracers.

Reply:
Yes that means it becomes tracer registry.

"just don't have bugs" is not a feasible answer to the problem unfortunately.

Fortunately, most end users won't have to worry about it too much, just library developers. The only change for most end users is the name of the global object they need to create. Most users will never call any methods on the tracer registry manually.

Reply:
Thanks for explaining.

Reply:
@dyladan I had not yet corrected the tracer to tracerRegis

> @dyladan should this now be `module.exports = opentelemetry.getTracerRegistry();` ?
> Should the file be renamed to `tracer-registry.js` in this case?


@dyladan I had not yet made the above fix. Do you want a follow up PR for this?

Reply:
You can make a follow up PR to handle it, but you're not actually using the exported value in your index, so it shouldn't matter.